src/chapters/booleans-and-conditionals/nested-conditionals.rst

```diff
@@ -0,0 +1,68 @@
+Nested Conditionals
+===================
+
+We can write code with more complex branching behavior by combining conditionals, and in particular by nesting conditionals. Let's see how this works by tackling the following problem.
+
+.. admonition:: Example
+
+    Write code that prints different messages based on the value of a number variable. If the number is odd, print nothing. If it is even, print "EVEN!" If it is also positive print "POSITIVE!".
+
+Our first attempt at a solution might look like this:
+
+.. sourcecode:: js
+
+   let num = 7;
+
+   if (num % 2 === 0) {
+      console.log("EVEN!");
+   }
+
+   if (num > 0) {
+      console.log("POSITIVE!");
+   }
+
+**Output**
+
+::
+
+   POSITIVE!
+
+We find that the output is ``POSITIVE!``, even though 7 is odd and so nothing should be printed. This code doesn't work as desired because we only want to test for positivity when we already know that the number is even. We can enable this behavior by putting the second conditional *inside* the first.
+
+.. sourcecode:: js
+
+   let num = 7;
+
+   if (num % 2 === 0) {
+       console.log("EVEN!");
+
+       if (num > 0) {
+           console.log("POSITIVE!");
+       }
+   }
+
+Try this with a bunch of values (even, odd, positive, negative) to ensure it works as desired. Nice, huh?
+
+Notice that when we put a conditional inside another conditional, the body of the nested conditional is indented by two tabs, and not just one. This convention allows us to easily determine which code is part of which conditional, by looking at the indentation level.
+
+Check Your Understanding
+------------------------
+
+.. admonition:: Question
+
+   What is printed when the following code runs?
```

Comment:
I like this example, but I feel that either one more concept check or a deeper problem would be beneficial.

1) Follow this concept check with a second, similar task.
OR
2) Start the code with "let num = x;" and then ask the students to identify the output for a set of possible x values.  Ex: 0, 2, 7, -5.
OR
3) As written, the code will never print "odd".  Instead of asking, "What is the output," retain only the conditional and ask, "Under what circumstances will the program print 'odd'?"

Reply:
Good idea. I added  a second question here that expands on the first.